bookwyrm/templates/landing/login.html

```diff
@@ -6,7 +6,7 @@
 {% block content %}
 <h1 class="title">{% trans "Log in" %}</h1>
 <div class="columns is-multiline">
-    <div class="column is-half">
+    <div class="column {% if site.allow_registration %}is-half{% else %}is-full is-half-desktop{% endif %}">
```

Comment:
I decided that this works fine on tablets when registration is enabled to remain as half width, because then the other half is the registration form vs the about snippet (when it is disabled)

e.g.

![image](https://github.com/bookwyrm-social/bookwyrm/assets/36494925/2f6a6660-b175-4416-9c09-ae441395805b)